test/unit/resourceFilter.js

```diff
@@ -1,407 +1,199 @@
-'use strict'
+import assert from "assert";
+import { Filter } from "../../src/resource_filter";
 
-const assert = require('assert')
+import setupDb from "../integration/setup";
 
-describe('resourceFilter', () => {
-  const Filter = require('../../src/resource_filter')
-
-  describe('getExcluded', () => {
```

Comment:
I removed unit tests for `getExcluded` and `filterItem` because they are implementation details and now private methods.